languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -14776,6 +14776,21 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="SEEMING_SEEMS" name="confusion of is seeming/seems">
```

Comment:
Even though the pattern looks very specific, I still think we should use `default="temp_off"`, just to be sure it's tested first in the nightly test runs.